THIRD_PARTY_NOTICES

```diff
@@ -93,12 +94,15 @@ This project incorporates components with the licenses under which Camunda recei
   * com.nimbusds:lang-tag (Version: 1.6, License: Apache-2.0 (URL: https://opensource.org/licenses/Apache-2.0))
   * com.nimbusds:nimbus-jose-jwt (Version: 9.22, License: Apache-2.0 (URL: https://opensource.org/licenses/Apache-2.0))
   * com.nimbusds:oauth2-oidc-sdk (Version: 9.35, License: Apache-2.0 (URL: https://opensource.org/licenses/Apache-2.0))
+  * com.rabbitmq:amqp-client (Version: 5.14.2, License: ASL 2.0 (URL: no known URL), GPLv2 (URL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html), MPL 2.0 (URL: https://www.mozilla.org/en-US/MPL/2.0/))
```

Comment:
@chillleader or @markfarkas-camunda, can you please follow our [TPL guide](https://github.com/camunda/team-connectors/blob/main/how-to/THIRD_PARTY_LICENSES.md) to get rid of this?

Reply:
Removing the Spring Boot BOM dials this back so we're good for now. Plus, when using the Spring Boot BOM, it will be solved with https://github.com/camunda/connector-sdk/pull/315